catalog/service/rest/src/main/java/org/projectnessie/catalog/service/rest/IcebergErrorMapper.java

```diff
@@ -89,7 +89,7 @@ private static String message(BackendErrorStatus status, Throwable ex) {
   private IcebergErrorResponse mapStorageFailure(
       BackendErrorStatus status, Throwable ex, IcebergEntityKind kind) {
     // Log full stack trace on the server side for troubleshooting
-    LOGGER.info("Propagating storage failure to client: {}", status, ex);
+    LOGGER.debug("Propagating storage failure to client: {}", status, ex);
```

Comment:
My mistake on #8558 - I initially wanted to log only storage failures, but during various refactorings this `info` ended up covering Nessie and Iceberg exceptions too.

Would you mind still logging `INFO` for `!(NESSIE_ERROR | ICEBERG_ERROR)`? I think it helps debugging config mistakes a lot.